### ISSUE TITLE
Error when using tensor-split

### ISSUE THREAD

**Issue by ruped** [phronmophobic/llama.clj#1]
Just a heads up. When using `tensor-split` I get an error. 
```
  (def ctx (llama/create-context model-path {:n-gpu-layers 30
                                             :main-gpu 0
                                             :tensor-split [1.0]}))
```

I get an error:
```
1. Caused by java.lang.IllegalArgumentException
   Can not set com.sun.jna.ptr.FloatByReference field
   com.phronemophobic.llama.raw.structs.llama_context_params.tensor_split
   to [F

UnsafeFieldAccessorImpl.java:  167  jdk.internal.reflect.UnsafeFieldAccessorImpl/throwSetIllegalArgumentException
UnsafeFieldAccessorImpl.java:  171  jdk.internal.reflect.UnsafeFieldAccessorImpl/throwSetIllegalArgumentException
UnsafeObjectFieldAccessorImpl.java:   81  jdk.internal.reflect.UnsafeObjectFieldAccessorImpl/set
                Field.java:  780  java.lang.reflect.Field/set
            Structure.java:  668  com.sun.jna.Structure/setFieldValue
            Structure.java:  662  com.sun.jna.Structure/setFieldValue
            Structure.java:  837  com.sun.jna.Structure/writeField
                      REPL:   49  com.phronemophobic.llama/map->llama-params/fn
```

Not urgent. I'll take a look at solving this - but thought I would post this here too. Thanks!

**Comment by phronmophobic** [comment]
Good catch! That bug was introduced when updating the API.

**Comment by phronmophobic** [comment]
Should be fixed by c3391d5699fbf96fe63b214f1318a521f5cd9221.

**Comment by ruped** [comment]
Excellent. I've upgraded to the new commit and that fixes that for me. Thanks!
